CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project are documented in this file.
 
+## [unreleased]
+### Fixed
+- Fix python bindings compilation when toml is not installed (https://github.com/ami-iit/bipedal-locomotion-framework/pull/700)
```

Comment:
We never released a bugged version, so I am not sure if you want to include this in the CHANGELOG.

Reply:
the version 0.14.0 contains the bug :(

Reply:
https://github.com/ami-iit/bipedal-locomotion-framework/blob/2736a0467b867f2c0937f412f00c481223b4d3af/bindings/python/bipedal_locomotion_framework.cpp.in#L124-L126

Reply:
Ah right I was confusing this change with the CMake files change.